tests/testthat/helpers.R

```diff
@@ -24,8 +24,8 @@ library(ggplot2)
 # A helper function for testing: passes if the user looks at the plot and
 # presses enter, fails if the user enters anything else
 expect_manual_OK <- function(test.id, prompt="Look at the plot.") {
-  user.input <- readline(paste0("Test '",test.id, "': ", prompt, " Hit ENTER for OK; enter any character otherwise: "))
-  user.gives.OK <- user.input == ""
+  #user.input <- readline(paste0("Test '",test.id, "': ", prompt, " Hit ENTER for OK; enter any character otherwise: "))
+  user.gives.OK <- TRUE
   expect_true(user.gives.OK, label=paste0("Test '",test.id,"' okayed by user"))
 }
```

Comment:
nice short-term fix. in the long run, i think we should delete this whole function and all the calls to it - this `manual_OK` approach was interesting but is too clunky for automated testing. In the future we might look at what Hadley does for ggplot2 (https://github.com/hadley/ggplot2/tree/master/visual_test) and see if that would be a reasonable solution for us for testing plots.